websites/A/AsphaltEvents/presence.ts

```diff
@@ -0,0 +1,105 @@
+const presence = new Presence({
+    clientId: "751816190653104168"
+});
+
+let item,
+    item2;
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```

Comment:
move it above the item ones and merge the consts

Reply:
Resolved in updated pull request.